src/imagery/i.eodag/i.eodag.py

```diff
@@ -879,11 +879,7 @@ def skip_existing(output, search_result):
             if scene_file.exists():
                 creation_time = datetime.utcfromtimestamp(os.path.getctime(scene_file))
```

Comment:
```suggestion
                creation_time = str(
                    datetime.utcfromtimestamp(os.path.getctime(scene_file))
                )
```
Converting to string to be able to normalize it below...